src/apps/qutils/canvas.cpp

```diff
@@ -368,9 +368,11 @@ Geom::Size BaseCanvas::textBoundary(const std::string &text)
 }
 
 void BaseCanvas::transform(const Geom::AffineTransform &transform) {
-	painter.translate(transform.offset.x, transform.offset.y);
-	painter.rotate(180.0 * transform.rotate / M_PI);
-	painter.scale(transform.scale, transform.scale);
+	const auto &m = transform.m;
```

Comment:
use structure binding to simplify the code:
```
const auto& [m0, m1] = transform.m;
```

Reply:
ok